internal/policies/apparmor/apparmor_test.go

```diff
@@ -42,32 +42,30 @@ func TestApplyPolicy(t *testing.T) {
 		removeUnusedAssetsError bool
 		apparmorParserError     string
 
+		integrationTest bool
```

Comment:
ouch, I missed that, I think we should really stay away of integration tests at this level and try to mock every commands we are using (contrary to real integration tests). Do you think we can mock the parser and control its behaviour as we do with other commands rather?

Reply:
Hem, I don’t like we don’t understand that. Is it due to the parser talking with the kernel and this being denied? Do you mind having a look at this?

(Really, mixing integration tests here really strikes me as something we shouldn’t do for long term maintainance)

Reply:
phew! I really prefer that :)